packages/main/src/plugin/container-registry.ts

```diff
@@ -1679,17 +1679,22 @@ export class ContainerProviderRegistry {
     id: string;
     callback: (name: string, data: string) => void;
     abortController?: AbortController;
+    sinceDurationInSeconds?: number;
   }): Promise<void> {
     let telemetryOptions = {};
     let firstMessage = true;
     const container = this.getMatchingContainer(logsParams.engineId, logsParams.id);
+    const apiLogsParams: Dockerode.ContainerLogsOptions & { follow: true } = {
+      follow: true,
+      stdout: true,
+      stderr: true,
+      abortSignal: logsParams.abortController?.signal,
+    };
+    if (logsParams.sinceDurationInSeconds) {
+      apiLogsParams.since = `${logsParams.sinceDurationInSeconds}s`;
```

Comment:
follow-up of https://github.com/podman-desktop/podman-desktop/pull/12626/files#r2106896446, we should not contraint to seconds, why can't we just expose the since like other parameters? why forcing the input to be a number in seconds?

Reply:
The compat API doc for `since` looks incomplete to me when indicating only the timestamp.

The doc for the `docker container logs --since` specifies (https://docs.docker.com/reference/cli/docker/container/logs/):

```
The --since option shows only the container logs generated after a given date. You can specify the date as an RFC 3339 date, a UNIX timestamp, or a Go duration string (e.g. 1m30s, 3h)
```

I know that this doc is not 100% related to the API, but I tested in https://github.com/podman-desktop/podman-desktop/pull/12624 the format `since=70s` like proposed in this PR, and this format is correctly accepted.

Reply:
tested with Docker provider, this syntax is not recognized in this case (`strconv.ParseInt('...s'): invalid syntax

Reply:
I tried with `docker` CLI, if I use `--since 42m` it's providing the unix timestamp on the REST API


```
GET /v1.41/containers/ac3295a7b693b304dc1d2689608023e35f1307cd6d86452d7cea5b3cf03035a8/logs?since=1748258328&stderr=1&stdout=1&tail=all HTTP/1.1\r
```

and using `libpod` (not compat API)
```
GET /v5.5.0/libpod/containers/727c9186b31ec98bca45d0c246dbdeaa95580a8e57b5b438b653eac932cf476f/logs?follow=false&since=2025-05-26T13%3A21%3A44%2B02%3A00&stderr=true&stdout=true&tail=-1&timestamps=false&until=0001-01-01T00%3A00%3A00Z HTTP/1.1\r
```


so CLI uses minutes or seconds but when it's executing calls, it's giving an absolute date